v2/plugins/che-machine-exec-plugin/0.0.1/meta.yaml

```diff
@@ -0,0 +1,16 @@
+publisher: ""
```

Comment:
Should not plugins in `v2` folder contain non-empty publisher?

Reply:
I got why it is empty here

Reply:
Unfortunately we need this for the backward compatibility. Otherwise, after an update of wsmaster existing workspaces with plugins in old notation would not start.